<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AffichageClass</class>
 <widget class="QMainWindow" name="AffichageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Affichage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="lancerDe">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Lancer le d√©</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="deValeur">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="deTotal">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plusieurLancerStart">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Lancer le tirage</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>101</width>
      <height>261</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Lancer</string>
     </property>
    </column>
   </widget>
   <widget class="QSpinBox" name="plusieurLancerNombre">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuchoisir">
    <property name="title">
     <string>choisir</string>
    </property>
    <addaction name="actionde"/>
    <addaction name="actiondehisto"/>
   </widget>
   <addaction name="menuchoisir"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionde">
   <property name="text">
    <string>de</string>
   </property>
  </action>
  <action name="actiondehisto">
   <property name="text">
    <string>dehisto</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Affichage.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lancerDe</sender>
   <signal>clicked()</signal>
   <receiver>AffichageClass</receiver>
   <slot>lancerDe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reset</sender>
   <signal>clicked()</signal>
   <receiver>AffichageClass</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionde</sender>
   <signal>triggered()</signal>
   <receiver>AffichageClass</receiver>
   <slot>pageDe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actiondehisto</sender>
   <signal>triggered()</signal>
   <receiver>AffichageClass</receiver>
   <slot>pageDeHisto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plusieurLancerStart</sender>
   <signal>clicked()</signal>
   <receiver>AffichageClass</receiver>
   <slot>plusieurLancer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>lancerDe()</slot>
  <slot>reset()</slot>
  <slot>pageDe()</slot>
  <slot>pageDeHisto()</slot>
  <slot>plusieurLancer()</slot>
 </slots>
</ui>
